src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -989,6 +989,42 @@ let ``Uses documentation`` () =
     given ["-h"] |> yields message
     given ["-?"] |> yields message
 
+[<Fact>]
+let ``Shows help text for generateazurepayload command`` () =
+    let name = Path.GetFileNameWithoutExtension (Assembly.GetEntryAssembly().Location)
+    let message =
+        name
+        |> sprintf "Usage:
+                      %s generateazurepayload [options]
+
+                    Options:
+                      -n <n> (REQUIRED)                                   A number.
+                      --pauli <PauliI|PauliX|PauliY|PauliZ> (REQUIRED)    The name of a Pauli matrix.
+                      --my-cool-bool (REQUIRED)                           A neat bit.
+                      --target <target> (REQUIRED)                        The target device ID.
+                      --verbose                                           Show additional information about the submission.
+                      -?, -h, --help                                      Show help and usage information"
+    let given = test "Help"
+    given ["generateazurepayload"; "--help"] |> yields message
+
+[<Fact>]
+let ``Shows help text for generateazurepayload command with default target`` () =
+    let name = Path.GetFileNameWithoutExtension (Assembly.GetEntryAssembly().Location)
+    let message =
+        name
+        |> sprintf "Usage:
+                      %s generateazurepayload [options]
+
+                    Options:
+                      -n <n> (REQUIRED)                                   A number.
+                      --pauli <PauliI|PauliX|PauliY|PauliZ> (REQUIRED)    The name of a Pauli matrix.
+                      --my-cool-bool (REQUIRED)                           A neat bit.
+                      --target <target>                                   The target device ID.
+                      --verbose                                           Show additional information about the submission.
+                      -?, -h, --help                                      Show help and usage information"
```

Comment:
I have an impression of the code duplication for the fragment 
```f#
    let name = Path.GetFileNameWithoutExtension (Assembly.GetEntryAssembly().Location)
    let message =
        name
        |> sprintf "Usage:
                      %s generateazurepayload [options]
                    Options:
                      -n <n> (REQUIRED)                                   A number.
                      --pauli <PauliI|PauliX|PauliY|PauliZ> (REQUIRED)    The name of a Pauli matrix.
                      --my-cool-bool (REQUIRED)                           A neat bit.
                      --target <target>                                   The target device ID.
                      --verbose                                           Show additional information about the submission.
                      -?, -h, --help                                      Show help and usage information"
```

Reply:
Since these are test cases, I believe code duplication is ok because it is easier to see what is the expected message for any particular test case.